src/LinuxTracing/LinuxTracingIntegrationTest.cpp

```diff
@@ -377,6 +386,11 @@ void VerifyOrderOfAllEvents(const std::vector<orbit_grpc_protos::ProducerCapture
   uint64_t previous_event_timestamp_ns = 0;
   for (const auto& event : events) {
     switch (event.event_case()) {
+      case orbit_grpc_protos::ProducerCaptureEvent::kCaptureStarted:
```

Comment:
We could probably also test that it's sent at all.

Reply:
no longer part of LinuxTracingHandler